src/reducers/resourceValidationHelpers.js

```diff
@@ -88,6 +88,32 @@ export const literalDateTimeStampValidationError = (
   return errors
 }
 
+export const literalEdtfValidationError = (value, propertyTemplate) => {
+  const errors = []
+  const edtf = require("edtf")
```

Comment:
If this is causing a test issue, you could try importing the library like we do the other ones, by putting this line near the top of the file:

```javascript
import edtf from "edtf"
```

🤷🏻‍♂️

Reply:
import statement doesn't work;  there's something about how edtf module is written.  https://www.npmjs.com/package/edtf suggests

```
import edtf, { Date, Interval } from 'edtf'
```

but that won't work.   The require did work to get the test passing.

```
import { edtf } from 'edtf`
```

fails the test.